src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Yea, that's my point. In this case, we're trying to stub something that does not exist natively and, as far as we know, no one uses. We used a plain object before, but it seems to me that nothing we add would be a truly accurate stub, so we may as well `delete` it.